### PR TITLE
Implement new records

### PR DESCRIPTION
**Pull request by IvanCraft623**
## Introduction
Implement missing records.

### API changes
The following enum values has been added to `RecordType` class:
- `DISK_5`
- `DISK_OTHERSIDE`
- `DISK_PIGSTEP`

## Follow-up
Fix https://github.com/pmmp/BedrockProtocol/issues/115

## Tests
coming soon :D

### CODE REVIEW

File: src/block/utils/RecordType.php [hunk 1]
```diff
@@ -55,18 +58,20 @@ final class RecordType{
 	protected static function setup() : void{
 		self::registerAll(
 			new RecordType("disk_13", "C418 - 13", LevelSoundEvent::RECORD_13, KnownTranslationFactory::item_record_13_desc()),
+			new RecordType("disk_5", "Samuel Åberg - 5", 439, KnownTranslationFactory::item_record_5_desc()),
```
Comment (dktapps) [inline src/block/utils/RecordType.php]: Don't hardcode this. It makes it hard to spot when things are missing.

**Comment by dktapps** [comment]
BedrockProtocol has been updated, so you can now complete the PR.

**Comment by IvanCraft623** [comment]
> BedrockProtocol has been updated, so you can now complete the PR.

you have not bumped BedrockProtocol

**Comment by dktapps** [comment]
> > BedrockProtocol has been updated, so you can now complete the PR.
> 
> you have not bumped BedrockProtocol

Sorry, my mistake, I thought I'd already tagged it. Will take care of it later.

**Pull request merged**